app/presenters/two_factor_auth_code/backup_code_presenter.rb

```diff
@@ -0,0 +1,25 @@
+module TwoFactorAuthCode
+  class BackupCodePresenter < TwoFactorAuthCode::GenericDeliveryPresenter
+    include Rails.application.routes.url_helpers
+    include ActionView::Helpers::TranslationHelper
+
+    attr_reader :credential_ids
```

Comment:
Are `credential_ids` used anywhere in/by the presenter?

Reply:
done